### ISSUE TITLE
The test suite is broken

### ISSUE THREAD

**Issue by brunoarueira** [startae/uploadbox#13]
Hey,

The test suite is broken. To help on fix that, what do you think about add to [travisci](https://travis-ci.org) to run the entire suite and fix each test?

**Comment by renatocarvalho** [comment]
@brunoarueira That would be great. Would you have some time to work on this?

One important thing to mention is that we're considering dropping Uploadbox and using https://github.com/thoughtbot/paperclip. It's been a while since we don't touch this and the code is quite complex. 

What do you think?

**Comment by brunoarueira** [comment]
About add to travisci, I can make a pr add .travis.yml, but it is in your end to configure on travisci.org.

And about dropping uploadbox, depends on your needs, from what I've used in the past it is all about carrierwave or refile.

I have found this blog post (https://infinum.co/the-capsized-eight/articles/best-rails-image-uploader-paperclip-carrierwave-refile) and maybe it clarifies a little bit about this gems.
